RCAIDE/Library/Plots/Thermal_Management/plot_air_cooled_conditions.py

```diff
@@ -48,16 +48,13 @@ def plot_air_cooled_conditions(air_cooled, results, coolant_line, save_figure,sh
     # get line colors for plots 
     line_colors   = cm.inferno(np.linspace(0,0.9,len(results.segments)))     
 
-    fig = plt.figure(save_filename)
-    fig.set_size_inches(width,height) 
-    axis_0 = plt.subplot(1,1,1)
-    axis_1 = plt.subplot(2,1,1)
-    axis_2 = plt.subplot(2,1,2)        
-    b_i = 0 
-   
-    axis_0.plot(np.zeros(2),np.nan*np.zeros(2), color = line_colors[0], marker = ps.markers[b_i], linewidth = ps.line_width, label = air_cooled.tag) 
-    axis_0.grid(False)
-    axis_0.axis('off')  
+    fig_1 = plt.figure(save_filename + '_Effectivess')
+    fig_2 = plt.figure(save_filename + '_Heat_Removed')
+    fig_1.set_size_inches(width,height)  
+    fig_2.set_size_inches(width,height)  
+    axis_1 = plt.subplot(1,1,1)
+    axis_2 = plt.subplot(1,1,1)        
+    b_i = 0  
    
```

Comment:
Two Many figures will be created if the battery modules are not identical, it is not advisable to do this...

Reply:
That's ok. Most of the time they will be, in the event that they aren't, the user should see them